src/set.js

```diff
@@ -22,7 +22,7 @@ var keys = require('./keys');
  * @return {Object} The merged object.
  * @example
  *
- *      R.set({age: 40, {name: 'fred', age: 10});
+ *      R.set({age: 40, {name: 'fred', age: 10}});
```

Comment:
Let's fix this line, @paldepind, so the pull request is in a mergeable state. :)